src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -414,3 +414,44 @@ paths:
           description: Client error
         '403':
           description: Permission denied
+
+  /releases/{name}/pinnable:
+    parameters:
+      - $ref: '#/components/parameters/ReleaseName'
+
+    put:
+      summary: Set a release as pinnable on a channel
+      operationId: auslib.web.admin.views.releases_v2.set_pinnable
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - product
+                - channel
+                - version
+              properties:
+                product:
+                  type: string
+                channel:
+                  type: string
+                version:
+                  type: string
+                # Creates a scheduled change if passed
+                when:
+                  type: integer
+                  nullable: true
+      responses:
+        '200':
+          description: OK
+          content:
+            # TODO: check
```

Comment:
This looks right to me. Any reason to keep the TODO?

Reply:
@jcristau wrote that, I'll let him weigh in on if there is still something needed to be done there.

Reply:
Maybe just drop the comment if things works as-is, I think I put this in there because I hadn't actually tested the API so wasn't sure what the response content should be.